tests/bluetooth/host/buf/bt_buf_get_cmd_complete/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.20.0)
+
+if (NOT BOARD STREQUAL unit_testing)
```

Comment:
do we still have to do this after torstein's PR?

Reply:
I don't know what made this check redundant, but it seems not to be needed any more